kubernetes/operationcode_python_backend/overlays/prod/deployment.yaml

```diff
@@ -7,8 +7,7 @@ spec:
     spec:
       containers:
         - name: app
-          image: operationcode/back-end:master
-          imagePullPolicy: Always
```

Comment:
I think this should stay in.

Reply:
Okay since we are iterating we want to not pull always.

Reply:
That...doesn't make sense

Reply:
For now his development flow is to push things to master and manually deploy to staging or prod.